kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/wlst/createWLSDomain.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+#
+#Copyright (c) 2021, Oracle and/or its affiliates.
+#Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+#Adopted from https://github.com/oracle/docker-images/blob/main/OracleWebLogic/samples/12213-domain-home-in-image/container-scripts/createWLSDomain.sh
+#Define DOMAIN_HOME
+echo "Domain Home is: " $DOMAIN_HOME
+
+# If AdminServer.log does not exists, container is starting for 1st time
```

Comment:
The next 50 lines can be removed (they're all related to the unneeded admin server).